doc/source/ray-core/scheduling/memory-management.rst

```diff
@@ -274,6 +274,11 @@ In the above example, the memory quota is specified statically by the decorator,
   # override the memory quota to 1GiB when creating the actor
   SomeActor.options(memory=1000 * 1024 * 1024).remote(a=1, b=2)
 
+Ray reserves some amount of memory on each node for system operations, so the application memory available for task and actor scheduling will be less than the total memory on the node.
+By default, Ray reserves system memory equal to the object store capacity (default 30% of available memory).
+To adjust the amount of system memory that Ray should reserve, set the environment variable `RAY_system_memory_reservation_bytes` when starting each Ray node.
```

Comment:
Does changing `RAY_system_memory_reservation_bytes` affect the object store capacity?